src/RevivalPMMP/PEXProtection/Main.php

```diff
@@ -16,179 +16,206 @@
+                                    if(isset($args[3])) {
+                                        $this->allMobs = (strcmp(strtolower($args[3]), "true") == 0) ? true : false;
+                                    } else {
+                                        if ($this->getConfig()->exists("All-Mobs")) {
+                                            $this->allMobs = $this->getConfig()->get("All-Mobs");
+                                        } else {
+                                            $this->allMobs = true;
+                                        }
+                                    }
+                                } else {
+                                    $p->sendMessage(TF::RED .  "You have to define a radius for the protection center block!");
+                                }
+                            } else {
+                                $p->sendMessage(TF::RED . "A protection center block with that name already exists");
+                            }
+                            return true;
+                        
+	                    case "disableworld":
+	                    case "world":
+	                    	if(isset($args[1])) {
+			                    $worlds = $this->getConfig()->get("Disabled-Worlds");
+			                    $worlds[] = $args[1];
+			                    $this->getConfig()->set("Disabled-Worlds", $worlds);
+			                    $this->saveDefaultConfig();
+                                $p->sendMessage(TF::GREEN . "Successfully added a world to disable monster spawning.");
+		                    }
+		                    return true;
+	                    
+                        case "delete":
+                        case "del":
+                        case "remove":
+                        case "rem":
+                        case "clear":
+                            if($this->isCenterBlock($args[1])) {
+                                $this->centers->remove($args[1]);
+                                $this->centers->save();
+                                $p->sendMessage(TF::AQUA . "Successfully removed the protection center block " . $args[1] . "!");
+                            } else {
+                                $p->sendMessage(TF::RED . "That protection center block name does not exist.");
+                            }
+                            return true;
+                    }
+                } else {
+                    $p->sendMessage(TF::RED . "Please provide a valid protection center block name!");
+                }
+            } else {
+                $this->getServer()->getLogger()->info(TF::RED . "PEXProtector commands must be used by players.");
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * @param PlayerInteractEvent $ev
+     */
+    public function onInteract(PlayerInteractEvent $ev) {
+        $p = $ev->getPlayer();
+        if($this->inTapMode($p)) {
```

Comment:
what if the event is cancelled?

Reply:
Great question.  I didn't consider that. It will be updated shortly.